Formula/g/github-keygen.rb

```diff
@@ -6,6 +6,8 @@ class GithubKeygen < Formula
   license "GPL-3.0-or-later"
   head "https://github.com/dolmen/github-keygen.git", branch: "release"
 
+  uses_from_macos "perl"
+
```

Comment:
can you run `brew style --fix github-keychain` to fix the syntax issue?

Reply:
Not sure how I missed that. Apologies. Fixed now 👍